src/towncrier/test/test_project.py

```diff
@@ -45,32 +44,65 @@ def test_tuple(self):
         version = get_version(temp, "mytestproja")
         self.assertEqual(version, "1.3.12")
 
-    def test_incremental(self):
+        def test_incremental(self):
+            """
+            An incremental-like Version __version__  is picked up.
+            """
+            temp = self.mktemp()
+            os.makedirs(temp)
+            os.makedirs(os.path.join(temp, "mytestprojinc"))
+
+            with open(os.path.join(temp, "mytestprojinc", "__init__.py"), "w") as f:
+                f.write(
+                    """
+    class Version:
+        '''
+        This is emulating a Version object from incremental.
+        '''
+
+        def __init__(self, *version_parts):
+            self.version = version_parts
+
+        def base(self):
+            return '.'.join(map(str, self.version))
+
+
+    __version__ = Version(1, 3, 12, "rc1")
+    """
+                )
+            version = get_version(temp, "mytestprojinc")
+            self.assertEqual(version, "1.3.12rc1")
+
+    def test_not_incremental(self):
         """
-        An incremental Version __version__  is picked up.
+        An incremental-like Version __version__  is picked up, base takes unexpected
+        arguments.
```

Comment:
Maybe something like this ?


```suggestion
        An excpettion is raised when the version could not be detected.
        For this test we use an incremental-like object,
        that has the `base` method but that method
        does not matches the return type for `incremental`.
```

Reply:
```suggestion
        An exception is raised when the version could not be detected.
        For this test we use an incremental-like object,
        that has the `base` method, but that method
        does not match the return type for `incremental`.
```
fixed some typos